### PR TITLE
added top navbar with search and bootstrap

### PR DESCRIPTION
**Pull request by mlenahan**
left a bit of an explanation on slack there

### CODE REVIEW

File: ecommerce/products/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+</head>
+
+<nav class="navbar navbar-inverse">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <a class="navbar-brand" href="#">PokemonVillage</a>
```
Comment (johnpooch) [inline ecommerce/products/templates/base.html]: Would be good to do a `href="{% url 'home' %}"` here and get rid of the home nav item maybe?

File: ecommerce/products/templates/base.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          </button>
+        </div>
+      </div>
+    </form>
+  </div>
+</nav>
+
+<body>
+    {% block  content %}
+    replace me
```
Comment (johnpooch) [inline ecommerce/products/templates/base.html]: Bin this :P

**Review by johnpooch — CHANGES_REQUESTED** [review]

File: ecommerce/products/templates/base.html (hunk 1 above)
Comment (mlenahan) [reply ecommerce/products/templates/base.html]: Done

File: ecommerce/products/templates/base.html (hunk 2 above)
Comment (mlenahan) [reply ecommerce/products/templates/base.html]: Done :P

**Pull request merged**